wrappers/msvcenv.sh

```diff
@@ -31,6 +31,7 @@ SDKINCLUDE="$BASE\\$SDK\\include\\$SDKVER"
 SDKLIB="$BASE\\$SDK\\lib\\$SDKVER"
 BINDIR=$BASE_UNIX/vc/tools/msvc/$MSVCVER/bin/Hostx64/$ARCH
 SDKBINDIR=$BASE_UNIX/$SDK_UNIX/bin/$SDKVER/x64
+MSBUILDBINDIR=$BASE_UNIX/MSBuild/Current/Bin/amd64
```

Comment:
Note that while we hardcode the `x64` bindirs above, we overwrite this with a sed command in `install.sh`, see this snippet:
```
host=x64
if [ "$(uname -m)" = "aarch64" ]; then
    host=arm64
fi
cat $ORIG/wrappers/msvcenv.sh | sed 's/MSVCVER=.*/MSVCVER='$MSVCVER/ | sed 's/SDKVER=.*/SDKVER='$SDKVER/ | sed s/x64/$host/ > msvcenv.sh
```
So I'd appreciate if you'd update this accordingly to change the arch from amd64 to arm64 or whatever the corresponding directory is called here as well.

Reply:
And as a separate point: For various reasons, we lowercase many other directories, like `vc/tools/msvc/$MSVCVER/bin` above. But if we don't have any concrete reason for doing it here, we can certainly leave it in its original casing, and it's easy to revisit this if we run into issues with mixed casing and case sensitiveness.

Reply:
> So I'd appreciate if you'd update this accordingly to change the arch from amd64 to arm64 or whatever the corresponding directory is called here as well.

Done in new commit https://github.com/mstorsjo/msvc-wine/pull/103/commits/5a334b4371e28c943278c5f4466085a5c6c9efa0

Reply:
> And as a separate point: For various reasons, we lowercase many other directories, like `vc/tools/msvc/$MSVCVER/bin` above. But if we don't have any concrete reason for doing it here, we can certainly leave it in its original casing, and it's easy to revisit this if we run into issues with mixed casing and case sensitiveness.

As for this, I'm not sure. The original casing seems to work in WINE at least as far as `MSBuild.exe` is concerned. And other tools should not touch these directories.

Reply:
Yeah, this is probably fine - in the long run it's definitely better to alter the case as little as possible, only when specifically needed.

As long as things execute in wine, the case sensitiveness isn't usually an issue. It has been an issue with e.g. the output from `-showIncludes`, if it is used by a case sensitive tool to track dependencies for rebuilds though, but that probably doesn't matter here.